src/resources/views/crud/buttons/delete.blade.php

```diff
@@ -50,16 +35,11 @@ className: "bg-danger",
 		                    text: "{!! '<strong>'.trans('backpack::crud.delete_confirmation_title').'</strong><br>'.trans('backpack::crud.delete_confirmation_message') !!}"
 		                  }).show();
 
-			              // Hide the modal, if any
-			              $('.modal').modal('hide');
-
-			              // Remove the details row, if it is open
-			              if (row.hasClass("shown")) {
-			                  row.next().remove();
-			              }
+						// Hide the modal, if any
+						$('.modal').modal('hide');
 
 			              // Remove the row from the datatable
-			              row.remove();
+						  crud.table.row($("#crudTable a[data-route='"+route+"']").parents('tr')).remove().draw(false);
```

Comment:
```suggestion
			              crud.table.draw(false);
```

Since we are redrawing, we don't need to remove that row from the table ...

Reply:
Good one! Fixed